questdb-rs/src/ingress/mod.rs

```diff
@@ -55,6 +57,25 @@ use ring::{
     signature::{EcdsaKeyPair, ECDSA_P256_SHA256_FIXED_SIGNING},
 };
 
+/// Defines the maximum allowed dimensions for array data in binary serialization protocols.
+pub const MAX_ARRAY_DIMS: usize = 32;
+
+/// Line Protocol Version supported by current client.
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum LineProtocolVersion {
```

Comment:
Rename to `ProtocolVersion` to match what I think is a better config setting, I.e. `protocol_version=1|2|auto`.